### ISSUE TITLE
[SF][0.131.24][MessageBox][a11y] MessageBox with custom action needs to be descriptive

### ISSUE THREAD

**Issue by herlinsong** [SAP/ui5-webcomponents-react#2424]
Describe the bug
MessageBox with custom action needs to be descriptive

Isolated Example
https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--with-custom-action

To Reproduce

Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--with-custom-action
Scroll down and click on 'Open Messagebox' button to open MessageBox, tab to focus on "Custom Action" action

Expected behavior
The "Custom Action" action button needs to be add more description.

![image](https://user-images.githubusercontent.com/81735931/146484374-ce774e71-d9a5-43a3-9cd7-8de7e0a715da.png)

UI5 Web Components for React Information
@ui5/webcomponents version: 0.31.24
@ui5/webcomponents-react version: 0.131.24
Operating System: macOS Monterey
Browser: Chrome Version 96.0.4664.93 (Official Build) (x86_64)

**Comment by Lukas742** [comment]
Hi @herlinsong 

I don’t really understand what you mean with “add more description”. Please elaborate.

**Comment by herlinsong** [comment]
Hi @Lukas742 ,

![image](https://user-images.githubusercontent.com/81735931/146708962-3961488b-41ae-4b17-a512-5953545c37ad.png)

For example, the above screenshot, when focus on the "Add" button, the screen reader announcement should be “Add this role”. At the moment, we cannot config the aria-label of the action button of the MessageBox.

**Comment by Lukas742** [comment]
This feature is also included in [v0.131.28](https://github.com/SAP/ui5-webcomponents-react/blob/rel/0.131/CHANGELOG.md#013128-2021-12-21).
